examples-next/basic/schema.ts

```diff
@@ -229,7 +227,7 @@ export const extendGraphqlSchema = graphQLSchemaExtension({
       createRandomPosts(root, args, context) {
         // TODO: add a way to verify access control here, e.g
         // await context.verifyAccessControl(userIsAdmin);
-        const lists: KeystoneListsAPI<KeystoneListsTypeInfo> = context.lists;
+        const { lists } = context;
```

Comment:
Why have we dropped the usage of the generated types here?

Reply:
Well `lists` is already typed as it comes of the `context` object. The re-typing serves to restrict it (there are a couple of fields that are not included in the generated types, in particular `search` and `orderBy`). This gives it the more correct typing. If we want to restrict usage of `search` and `orderBy` completely then we should do it at the `context` type layer.

I've seen a few examples like there where we have slightly inconsistent types, I'm trying to grind through the backlog and get everything talking the same language.

Reply:
Just FYI, I don't believe `search` and `orderBy` were deliberately restricted, they were probably just missed in the first pass of types.

Reply:
> Just FYI, I don't believe search and orderBy were deliberately restricted, they were probably just missed in the first pass of types.

orderBy was deliberately excluded since it should really be used anymore though if people want to add it, that's fine(though probs add an `@deprecated`)